code/game/objects/items/toys.dm

```diff
@@ -567,15 +567,6 @@
 	attack_verb = list("boned", "dunked on", "worked down to the bone")
 	chattering = TRUE
 
-	var/list/papyrus_messages = list(
-		"That's the disposal bin. Feel free to visit it at any time.",
-		"I can't just let anyone ERP with me, I'm a skeleton with standards!",
-		"Sorry. Can't talk. I'm busy being popular on a video game.",
-		"You can't spell 'robust' without several letters from my name!!!")
-	var/list/sans_messages = list(
-		"You feel your redtext crawling on your back.",
-		"On days like these, references like this should be BURNING IN HELL.",
-		"I've gotten a ton of work done today. A skele-ton.")
```

Comment:
Spoiler, some of the regular lines are undertale references, but don't tell Goof, he'll go crazy trying to work out which ones they are.